assembl/static2/css/views/synthesis.scss

```diff
@@ -1,3 +1,18 @@
+.syntheses-page {
+  .syntheses-no-synthesis {
+    font-size: 20px;
+    margin-top: 94px;
```

Comment:
why having a non rounded margin (94px vs 95px) ?

Reply:
k you prefer 100px